src/features/user/Account/payments.d.ts

```diff
@@ -1,4 +1,4 @@
-declare namespace Payments {
+export declare namespace Payments {
```

Comment:
1. Namespaces do not need to be exported.
2. It may make sense to remove the namespace declaration, and use regular import/export instead. If you make this change, please update the other files that use the Payments namespace.

https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-namespace.md

Reply:
@sunilsabatp Still to be removed in a few places.

![Screenshot 2023-03-10 at 2 51 13 PM](https://user-images.githubusercontent.com/44917347/224276515-8d267e15-1231-478b-b7d5-43b18c41c068.png)
<img width="1440" alt="Screenshot 2023-03-10 at 2 51 13 PM (2)" src="https://user-images.githubusercontent.com/44917347/224276520-5ad64796-5d0f-435f-9fa8-6993fdee7e7a.png">